.buildkite/pipeline.yml

```diff
@@ -156,6 +156,8 @@ steps:
 
   - label: ":hammer: Trigger Runtime FE Tests"
     trigger: "runtime-fe-end-to-end-tests"
+    build:
+      branch: dominik/run-3006-chromium-object-missing-properties
```

Comment:
Remove, pls.

Reply:
I will once we get FE tests to give us a good run :D